### DISCUSSION TITLE
How do should you handle p-values of zero?

### DISCUSSION THREAD

**Discussion by emilyjunkins** [adw96/breakaway#103] [Q&A]
Me again,

Below is the some output from this model from running `breakaway()`:
`betta(chats = data$estimate,
             ses = data$error,
             X = model.matrix(~ Dilution*Condition*Day))`

<img width="770" alt="Screen Shot 2021-01-27 at 11 54 58 AM" src="https://user-images.githubusercontent.com/23660076/106032766-82c15000-6096-11eb-8b95-daa29434481c.png">

1. Does `0.000` mean impossible under the null? 

2. How would I report the p-value and discuss significance?

**Comment by ailurophilia** [comment]
Hi Emily, 

Thanks for your question! 0 here does not mean impossible under the null – it means that the p-value is small enough that it's being rounded to zero, at least in this view of the model output. Since you have three digits after the decimal point in the p-value column, I would interpret these 0 values as an indication that p < 10^(-3), which is a reasonable way to report p-values in this situation. I hope this helps!

Best,
David
